README.md

```diff
@@ -54,6 +54,10 @@ Features that we do not currently support, but we do plan to add in the future:
 
 - To run the example app, you need an Ably API key. Create a free account on [ably.com](https://ably.com/) and then use your API key from there in the example app.
 - Clone the project
+- For Android, a Firebase instance configuration is required to build the app. In order run the application on Android device:
+  - Create a Firebase project following [official Firebase guide](https://firebase.google.com/docs/android/setup#create-firebase-project) (you can also use an existing one)
+  - Register the sample app with your Firebase project using `io.ably.flutter.example` as package name
+  - Download `google_services.json` file from Firebase and put it into [example/android/app](example/android/app) directory
```

Comment:
There is a whole other Readme for push notifications here https://github.com/ably/ably-flutter/blob/main/PushNotifications.md
I think PN related documentation is all covered there. I would argue that we should aim for a much shorter Readme for this repo and move PN related documentation to here, but probably something for the future

Reply:
The Firebase setup is described more in-depth in the `PushNotifications.md` file, but I thought it would be nice to describe the minimum required to run the app in the base documentation as well, because when I tried to run the sample, even without push notifications, the build failed because of missing Firebase configuration.

Reply:
Ok, I do agree with that (minimal information to run the app on main readme file). There is some work we will later do on push notifications :)